pkg/apiserver/debugapi/endpoint_def.go

```diff
@@ -0,0 +1,521 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package debugapi
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/pingcap/kvproto/pkg/metapb"
+
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/debugapi/endpoint"
+	"github.com/pingcap/tidb-dashboard/pkg/apiserver/model"
+)
+
+var pprofKindsParam = &endpoint.APIParam{
+	Model: endpoint.APIParamModelEnum([]endpoint.EnumItem{
+		{Name: "allocs"},
+		{Name: "block"},
+		{Name: "cmdline"},
+		{Name: "goroutine"},
+		{Name: "heap"},
+		{Name: "mutex"},
+		{Name: "profile"},
+		{Name: "threadcreate"},
+		{Name: "trace"},
```

Comment:
Maybe better to call them "Value" (and let the missing name to be filled with value).

Reply:
That makes sense. I went the opposite way.